filters/PMFFilter.cpp

```diff
@@ -88,14 +85,15 @@ void PMFFilter::addArgs(ProgramArgs& args)
 {
     args.add("cell_size", "Cell size", m_args->m_cellSize, 1.0);
     args.add("exponential", "Exponential growth of window size?",
-        m_args->m_exponential, true);
+             m_args->m_exponential, true);
     args.add("ignore", "Ignore values", m_args->m_ignored);
-    args.add("initial_distance", "Initial distance",
-        m_args->m_initialDistance, 0.15);
-    args.add("last", "Consider last returns only?", m_args->m_lastOnly, true);
+    args.add("initial_distance", "Initial distance", m_args->m_initialDistance,
+             0.15);
+    args.add("returns", "Include only returns?", m_args->m_returns,
+             {"last", "only"});
```

Comment:
It seems best to either validate this in initialize() or convert automatically using something like ReturnType, where you define ReturnType to encode the possible values.  There isn't a great way to do the second option right now with ProgramArgs, but I'm in the process of adding something.

Reply:
I'm going to test after lunch.